packages/yarnpkg-builder/sources/tools/makeConfig.ts

```diff
@@ -31,6 +31,28 @@ export function findTsconfig() {
   return npath.join(currTsContextRoot, `tsconfig.json`);
 }
 
+// We don't want to add the BuildPlugin outside of our infra.
+function getBuildPlugin() {
+  if (!process.env.BUILD_MONITORING_ENABLED)
+    return [];
+
+
+  const BuildPlugin = require(`@datadog/build-plugin/dist/webpack`).BuildPlugin;
+  return [new BuildPlugin({
+    context: npath.join(process.cwd(), `../..`),
+    datadog: {
+      apiKey: process.env.DD_API_KEY,
+      prefix: `webpack`,
+      tags: [
+        `branchname:${process.env.CIRCLE_BRANCH || `branch`}`,
+        `sha:${process.env.CIRCLE_SHA1 || `local`}`,
+        `jobname:${process.env.CIRCLE_JOB || `job`}`,
```

Comment:
```suggestion
        `branchname:${process.env.GITHUB_REF || `branch`}`,
        `sha:${process.env.GITHUB_SHA || `local`}`,
        `jobname:${process.env.GITHUB_EVENT_NAME || `job`}`,
```
https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables

Reply:
I don't know why I thought this build was running on CircleCI 🤦